### ISSUE TITLE
setData() getData() leads to inconsistencies with Number.MAX_SAFE_INTEGER

### ISSUE THREAD

**Issue by arlac77** [Symatem/SymatemJS#5]
```js
backend.setData(symbol, Number.MAX_SAFE_INTEGER)

Number.MAX_SAFE_INTEGER != backend.getData(symbol)
```

**Comment by Lichtso** [comment]
In consistency with WASM the JS values are currently limited to 32 bit as well.
So 4294967295 is the MAX of natural numbers for now.

**Comment by Lichtso** [comment]
Fixed by #6
